cmd/jimmctl/cmd/auth.go

```diff
@@ -6,15 +6,16 @@ import (
 	jujucmd "github.com/juju/cmd/v3"
 )
 
-var authDoc = `
-auth enables users to manage authorisation model used by JIMM.
+const authDoc = `
+The auth command enables users to manage authorisation model used by JIMM.
```

Comment:
this is/was a bit vague/misleading. We manage permissions/tuples, while the authorisation model is fixed (created by the charm).

Reply:
I'll update it.